### PR TITLE
Implement temperature scaling in PyLaia

### PR DESCRIPTION
**Pull request by MGenerali**
Temperature scaling is a special case of Platt scaling used to calibrate confidence scores. 
The logits are divided by a scalar T before being plugged into the softmax.

The main idea is explained in  https://arxiv.org/pdf/1706.04599.pdf

### CODE REVIEW

**Comment by starride-teklia** [comment]
Closing as #67 is merged
